GH#54809: Updates to Network Observability (#55076)
openshift/openshift-docs: modules/network-observability-loki-install.adoc

.Prerequisites

* Supported Log Store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
* {product-title} 4.10+.
* Linux Kernel 4.18+.

//* <Any Loki install prerequisites for using with Network Observability operator?>

> There are several ways you can install Loki. One way is by installing the Loki Operator using the {product-title} web console Operator Hub. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 

Author: Instead of having this listed as a prerequisite for Installing Loki, it makes more sense to have the recommendation here. In the Prerequisites, it doesn't make sense to have "Installed Loki" when the user is about to install Loki.

> There are several ways you can install Loki. One way is by installing the Loki Operator using the {product-title} web console Operator Hub. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], in which you can create a LokiStack instance using `openshift-network` configuration mode. It provides fully automatic, in-cluster authentication and authorization support for the Network Observability Operator. 

Reviewer: ```suggestion
There are several ways you can install Loki. One way you can install the Loki Operator is by using the {product-title} web console Operator Hub. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6]. This version provides the ability to create a LokiStack instance using `openshift-network` configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for the Network Observability Operator. 
```

Author: Agree, addressed :)

> There are several ways you can install Loki. One way is by installing the Loki Operator using the {product-title} web console Operator Hub. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], in which you can create a LokiStack instance using `openshift-network` configuration mode. It provides fully automatic, in-cluster authentication and authorization support for the Network Observability Operator. 

Author: I think yes makes perfect sense.

.. Verify that *Loki Operator* is listed with *Status* as *Succeeded* in all the projects.
+
. Create a `Secret` YAML file. You can create this secret in the web console or CLI. 

> .. Using the web console, navigate to the *Project: All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.

Reviewer: I haven't seen the `*Projects: All Project*` part be formatted this way in other parts of the docs when referencing the web console. I think even in this context it would be:
`*Projects* -> * All Projects*`

Author: You're right- this is a mistake on my part.

+
. Create a `Secret` YAML file. You can create this secret in the web console or CLI. 

> .. Using the web console, navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. Use the `access_key_id` and `access_key_secret` to specify your credentials. 

Author: I think that this information needed to be removed after [this](https://github.com/openshift/openshift-docs/pull/53263#discussion_r1054026176) discussion in the original PR where we discussed removing the step of using `oc create` to create the `netobserv` namespace.
